docs/runbooks/VirtOperatorDown.md

```diff
@@ -6,7 +6,7 @@
 
 This alert fires when no `virt-operator` pod in the `Running` state has been detected for 10 minutes. 
 
-The `virt-operator` is the first operator to start in a cluster. Its primary responsibilities include the following: 
+The `virt-operator` is the first Operator to start in a cluster. Its primary responsibilities include the following: 
 
 - Installing, live updating, and live upgrading a cluster
 
```

Comment:
Just spotted a typo under Impact: "custom workloads". That was probably supposed to be customer workloads.

Reply:
Well, I'm not that well versed in the terminology, but both seem like legitimate phrases. In addition, if this is upstream, can you actually talk about "customers"?

Still, if this is incorrect, it would have to be fixed in a couple of other PRs IIRC... >_<

Reply:
"custom workloads" doesn't make sense. It's just a workload, a bunch of VMs. I would probably say just "workloads" or VM workloads.

Reply:
Fair enough, should be fixed now.